go/tools/testwrapper/wrap_test.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"testing"
+)
+
+func TestShouldWrap(t *testing.T) {
```

Comment:
Separately, it would be good to have a `go_bazel_test` in `//tests/core/go_test` that invokes `bazel test` on a test target and verifies the output XML. That could be instead of or in addition to `xml_test.go`.

Reply:
Added it as an addition. The integration test under `//tests/core/go_test` is a little more tricky to completely verify as the test runtime will be different each execution, so I think this unittest is useful to test the explicit conversion from the json events to xml.